docs/conf.py

```diff
@@ -52,6 +52,7 @@
 intersphinx_mapping = {
     "https://docs.python.org/": None,
     "requests": ("http://docs.python-requests.org/en/master/", None),
+    "pandas": ("https://pandas.pydata.org/pandas-docs/stable", None),
```

Comment:
can you provide a screenshot showing that links to `pandas` docs from our docs work?

Reply:
<img width="351" alt="Screen Shot 2019-08-12 at 9 49 12 AM" src="https://user-images.githubusercontent.com/12500246/62869865-9633fc00-bce6-11e9-9415-7dc7f1605e7a.png">